__tests__/misc-api.test.js

```diff
@@ -0,0 +1,16 @@
+const request = require('supertest');
+
+const app = require('../app.js');
+const db = require('../db/connection.js');
+const seed = require('../db/seeds/seed.js');
+const testData = require('../db/data/test-data/');
+
+beforeEach(() => seed(testData));
+
+afterAll(() => db.end());
+
+describe("Misc. api tests", () => {
+  test("Returns 404 when requesting a non-existent endpoint", () => {
+    return request(app).get('/api/banana').expect(404);
```

Comment:
Seeing as you are testing for a custom error being returned by a non-existent endpoint, (you don't necessarily need this as express can do this by default) it might be best to test for a error message being returned, not just the status code (you may just be receiving express' 404 default error code)